packages/accordions/src/elements/accordion/Accordion.tsx

```diff
@@ -31,37 +31,39 @@ interface IStaticAccordionExport<T, P>
 }
 
 interface IAccordionProps extends Omit<HTMLAttributes<HTMLDivElement>, 'onChange'> {
-  /** The `aria-level` used to apply heading rank on accordion headers */
+  /** Sets `aria-level` heading rank in the document structure */
   level: number;
-  /** Sets the sections for expansion */
+  /** Sets the expanded sections in a controlled accordion */
   expandedSections?: number[];
-  /** Hides the bottom border under each accordion panel */
+  /** Hides section borders */
   isBare?: boolean;
-  /** Collapses panels in an uncontrolled accordion */
+  /** Allows uncontrolled accordion sections to collapse */
   isCollapsible?: boolean;
-  /** Reduces the padding on the accordion headers and panels */
+  /** Applies compact styling */
   isCompact?: boolean;
-  /** Determines whether the accordion panels animate */
+  /** Animates section expansion and collapse */
```

Comment:
Similar to the question above, should we refer to this area as an "accordion panel"?